Sources/APIKit/Request.swift

```diff
@@ -108,7 +115,7 @@ public extension Request {
             throw RequestError.invalidBaseURL(baseURL)
         }
 
-        var urlRequest = URLRequest(url: url)
+        var urlRequest = URLRequest(url: url, cachePolicy: cachePolicy)
```

Comment:
https://developer.apple.com/documentation/foundation/urlrequest/2011457-init
default value is `useProtocolCachePolicy `
It will not be degrade